torchrl/trainers/helpers/envs.py

```diff
@@ -428,6 +446,78 @@ def get_stats_random_rollout(
     return stats
 
 
+def initialize_observation_norm_transforms(
+    proof_environment: EnvBase,
+    num_iter: int = 1000,
+    key: Union[str, Tuple[str, ...]] = None,
+):
+    """Calling init_stats on all uninitialised ObservationNorms transform of a TransformedEnv.
```

Comment:
Maybe `Calls :obj:`ObservationNorm.init_stats` on all uninitialized :obj:`ObservationNorm` instances of a :obj:`TransformedEnv`.